src/components/Button/PlayButton.js

```diff
@@ -1,27 +1,25 @@
 import React, { PropTypes } from 'react';
+import styled, { css } from 'styled-components'
 
-/**
+const commonPlayBtnClasses = 'fa fa-play w3 h3 f3 absolute z-1 gray items-center justify-center br-pill pointer'
+const hoverPlayBtnClasses = `${commonPlayBtnClasses} bg-white-70`
+const playBtnClasses = `${commonPlayBtnClasses} hover-turquoise bg-white`
 
-.card-play-btn {
+const StyledPlayButton = styled.div`
```

Comment:
This is the first way to use `styled-components`. It creates a `div` that is styled with the css below. *Note that there are not quotes around any of the CSS.*